element wise operators support dynamic shape input and broadcast

uni19 opened the pull request:
# Description

## Type of change
- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

peri044 commented:
@uni19 Linter failed for me. Can you fix it ? Reference: [Linter](https://github.com/NVIDIA/TRTorch/blob/master/CONTRIBUTING.md#coding-guidelines)
But the PR looks good. Thanks for the contribution.

uni19 commented:
> @uni19 Linter failed for me. Can you fix it ? Reference: [Linter](https://github.com/NVIDIA/TRTorch/blob/master/CONTRIBUTING.md#coding-guidelines)
> But the PR looks good. Thanks for the contribution.

I run `bazel //tools/linter:cpplint -- //...`, these two submitted files keep not changed.  But these following files are modified:

        modified:   core/conversion/converters/impl/activation.cpp
        modified:   core/conversion/converters/impl/plugins/interpolate_plugin.cpp
        modified:   core/conversion/converters/impl/reduce.cpp
        modified:   core/conversion/converters/impl/select.cpp
        modified:   core/conversion/converters/impl/shuffle.cpp
        modified:   core/conversion/evaluators/aten.cpp
        modified:   core/conversion/evaluators/prim.cpp
        modified:   cpp/api/include/trtorch/ptq.h

narendasan commented:
@uni19 not sure why it got messed up when i tried to rebase but it looks good to me functionality wise, I have put your commits in a new PR ( #237) with the proper linting and will merge that. Thanks for the contribution and sticking with this!
